bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -418,6 +418,7 @@ spec:
                 - --health-probe-bind-address=:8081
                 - --metrics-bind-address=127.0.0.1:8080
                 - --leader-elect
+                - --logLevel 4
                 command:
                 - /manager
                 image: quay.io/etirelli/opendatahub-operator:dev-0.0.6
```

Comment:
Not sure when this was introduced , but this is causing CI test failures. This should be `REPLACE_IMAGE:latest`

Reply:
yeah, i have a fix for this https://github.com/opendatahub-io/opendatahub-operator/pull/388

Reply:
Sorry, I missed that. Thanks for fixing.